I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00action_file.rpy", line 415, in __call__
    renpy.save(fn, extra_info=save_name)
TypeError: renpy.loadsave.save() argument after ** must be a mapping, not tuple

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in script
    $ ui.interact()
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in <module>
    $ ui.interact()
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\core.py", line 3286, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\screen.py", line 791, in event
    rv = self.child.event(ev, x, y, st)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\layout.py", line 1526, in event
    rv = super(Window, self).event(ev, x, y, st)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\layout.py", line 285, in event
    rv = d.event(ev, x - xo, y - yo, st)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\behavior.py", line 1182, in event
    return handle_click(self.clicked)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\behavior.py", line 1103, in handle_click
    rv = run(action)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\behavior.py", line 394, in run
    new_rv = run(i, *args, **kwargs)
  File "D:\Python\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\display\behavior.py", line 401, in run
    return action(*args, **kwargs)
  File "renpy/common/00action_file.rpy", line 415, in __call__
    renpy.save(fn, extra_info=save_name)
  File "functions.py", line 131, in new_save
TypeError: renpy.loadsave.save() argument after ** must be a mapping, not tuple

Windows-10-10.0.22621 AMD64
Ren'Py 8.3.4.24120703
 
Sat Feb  8 12:30:40 2025
